common-labs/data-actions/data-actions.md

```diff
@@ -0,0 +1,183 @@
+# Data Actions
+
+## Introduction
+
+In this lab, you will build two simple canvases and learn how to create data actions to pass values between different canvases in a workbook.
+
+  ![Data actions overview](images/data-actions-overview.png)
+
+Estimated Time: __ minutes
```

Comment:
Please fill out how long this lab will take.

Reply:
Done!